cmd/agent/gui/checks.go

```diff
@@ -134,17 +147,19 @@ func reloadCheck(w http.ResponseWriter, r *http.Request) {
 // Sends the specified config (.yaml) file
 func getCheckConfigFile(w http.ResponseWriter, r *http.Request) {
 	fileName := mux.Vars(r)["fileName"]
-	paths := getAllPaths("yaml")
+
+	// Check if this file is in a subdir (file name is of the form `{check name}: {file name}`)
+	if i := strings.Index(fileName, ":"); i != -1 {
+		checkName := fileName[:i]
+		fileName = checkName + ".d/" + strings.TrimPrefix(fileName, checkName+": ")
```

Comment:
I don't think `.d/` will work on windows. We should use `filepath.Join`, I guess.